docs/Reference/Permissioning-Types.md

```diff
@@ -0,0 +1,71 @@
+---
+description: Permissioning types reference
+---
+
+# Permissioning types
+
+This reference describes account access and status types for
+[permissioning](../Concepts/Permissioning/PermissionsOverview.md).
+
+## Account access types
+
+The following table indicates the numeric value for each account access type.
+
+|   AccessType   | Value |
+|:--------------:|:-----:|
+|    ReadOnly    |   0   |
+|    Transact    |   1   |
+| ContractDeploy |   2   |
+|   FullAccess   |   3   |
+
+When setting the account access, the system checks if the account setting the access has
+sufficient privileges to perform the activity.
+
+* Accounts with `FullAccess` can grant any access type (`FullAccess`, `Transact`, `ContractDeploy`
+  or `ReadOnly`) to any other account.
+
+* Accounts with `ContractDeploy` can grant only `Transact`, `ContractDeploy` or `ReadOnly` access to other accounts.
+
+* Accounts with `Transact` access can grant only `Transact` or `ReadOnly` access to other accounts.
+
+* Accounts with `ReadOnly` access cannot grant any access.
+
+### Organization status types
+
+The following table indicates the numeric value for each organization status.
```

Comment:
Are there any defined transitions between status? If so, maybe you could display it using [Plant UML diagrams](https://github.com/ConsenSys/doc.common/wiki/MkDocs-And-Custom-Markdown-Guide#plant-uml) as a [state diagram](https://plantuml.com/state-diagram).